__tests__/app.test.js

```diff
@@ -159,12 +159,65 @@ describe("/api/articles/:article_id/comments", () => {
       .then((response) => {
         expect(response.body.err).toBe("Article does not exist");
       });
-  })
-  it("Responds with 200 and empty array if no comments",()=>{
+  });
+  it("Responds with 200 and empty array if no comments", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.comments).toEqual([]);
+      });
+  });
+});
+
+describe("POST /api/articles/:article_id/comments", () => {
+  it("returns 201 and correct comment body when all details correct ", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "my comment is here",
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(newComment)
+      .expect(201)
+      .then((comment) => {
+        const addedComment = comment.body.addedComment.body;
+        expect(newComment.body).toEqual(addedComment);
+      });
```

Comment:
What other properties do we need to be included in our response body? We don't just want the username and body, we want the entire new comment object

Reply:
Good changes here, I would suggest that we know what our comment_id is going to be as it auto-increments, so check for an exact number rather than any number